frontend/__tests__/unit/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { Breadcrumbs } from 'components';
+import { render } from '@testing-library/react';
+
+describe('Breadcrumbs', () => {
+  describe('when pathname is "/"', () => {
+    it('should render a single link to the home page', () => {
+      const element = render(<Breadcrumbs pathname="/" />);
```

Comment:
TypeError: Property 'labels' is missing in type '{ pathname: string; }' but required in type 'BreadCrumbProps'.